video_xblock/static/js/studio-edit.js

```diff
@@ -226,8 +226,15 @@ function StudioEditableXBlock(runtime, element) {
             }
         })
     };
-    /** Replaces an existing transcript to transcriptsValue or adds new */
-    var pushTranscript = function (lang, label, url, oldLang, $uploadButton){
+    /**
+     * Replaces an existing transcript to transcriptsValue or adds new
+     * Returns true if new one was added or false otherwise
+     * @param {String} lang
+     * @param {String} label
+     * @param {String} url
+     * @param {String} oldLang
+     */
+    var pushTranscript = function (lang, label, url, oldLang){
```

Comment:
var pushTranscript = function pushTranscript(lang, label, url, oldLang) 
add name to function